scripts/cfgToDot.scala

```diff
@@ -0,0 +1,37 @@
+import gremlin.scala._
+import io.shiftleft.codepropertygraph.generated._
+import java.nio.file.Paths
+import io.shiftleft.queryprimitives.utils.ExpandTo
+import org.apache.tinkerpop.gremlin.structure.Direction
+import io.shiftleft.queryprimitives.steps.Implicits.JavaIteratorDeco
+import javax.script.ScriptEngineManager
+
+
+/** Some helper functions: adapted from ReachingDefPass.scala in codeproperty graph repo */
+def vertexToStr(vertex: Vertex): String = {
+  try {
+    val methodVertex = vertex.vertices(Direction.IN, "AST").nextChecked
+    val fileName = ExpandTo.methodToFile(methodVertex) match {
```

Comment:
I never really got why we have this `ExpandTo` class. My guess is performance. If possible, let's not use it. I'd say though, let's just add a TODO for now to remind us that we should use the DSL instead `ExpandTo` - and merge this.